v03_pipeline/lib/tasks/reference_data/updated_reference_dataset_collection_task.py

```diff
@@ -0,0 +1,56 @@
+import hail as hl
+import luigi
+
+from v03_pipeline.lib.model import ReferenceDatasetCollection
+from v03_pipeline.lib.paths import valid_reference_dataset_collection_path
+from v03_pipeline.lib.reference_data.dataset_table_operations import (
+    update_or_create_joined_ht,
+)
+from v03_pipeline.lib.tasks.base.base_update_task import BaseUpdateTask
+from v03_pipeline.lib.tasks.files import GCSorLocalTarget
+
+
+class UpdatedReferenceDatasetCollectionTask(BaseUpdateTask):
+    reference_dataset_collection = luigi.EnumParameter(enum=ReferenceDatasetCollection)
+    dataset = luigi.OptionalStrParameter()
+
+    @property
+    def _destination_path(self) -> str:
+        return valid_reference_dataset_collection_path(
+            self.reference_genome,
+            self.dataset_type,
+            self.reference_dataset_collection,
+        )
+
+    def complete(self) -> bool:
```

Comment:
nit but I think just leaving the comment is fine.  the method itself, as written, is a no-op because of the inheritance.

Reply:
removed the comment in the code, for now it lives in my brain